data/libs/functions/functions.lua

```diff
@@ -1154,3 +1154,19 @@ function onDeathForDamagingPlayers(creature, func)
 		end
 	end
 end
+
+function string.toPosition(inputString)
```

Comment:
You need move for here: https://github.com/opentibiabr/canary/blob/main/data/libs/functions/string.lua